src/otaclient/configs/_cfg_configurable.py

```diff
@@ -58,6 +58,8 @@ class _OTAClientSettings(BaseModel):
     MAX_CONCURRENT_DOWNLOAD_TASKS: int = 128
     DOWNLOAD_INACTIVE_TIMEOUT: int = 5 * 60  # seconds
     MAX_RETRY_ON_ENTRY_COUNT: int = 300  # counts. almost 5 minutes
+    CLIENT_UPDATE_TIMEOUT: int = 5 * 60  # seconds
```

Comment:
Will be used to wait for sub ecus.

Reply:
Thank you. I don't have strong opinion to use either `5 * 60` or `300` :)
The existing code uses (min * 60) style, so let me keep the code consistency at here.
https://github.com/tier4/ota-client/blob/5f534536e36b502e3b7d4f4c84b6839a7fd53290/src/otaclient/configs/_cfg_configurable.py#L59

https://github.com/tier4/ota-client/blob/5f534536e36b502e3b7d4f4c84b6839a7fd53290/src/otaclient/configs/_cfg_configurable.py#L97